app/pages/health/routes.js

```diff
@@ -0,0 +1,20 @@
+import express from 'express';
+import { logger } from '../../logger';
+import { getReadyStatus } from './ready/getReadyStatus';
+import { getLiveStatus } from './live/getLiveStatus';
+
+const router = express.Router();
+
+router.get('/live', async (req, res) => {
+  logger.info('navigating to health/live page');
+  const status = await getLiveStatus();
```

Comment:
why can't you just assign it here instead of using a function to only return something?

Reply:
also doesn't need awaited

Reply:
I think it's so both live and ready go through the same layer to get the status. I'm going to keep is as now. Might be worth revisiting these later.

Reply:
I don't agree - ready has to call apis, this one should be a simple return. What is the benefit of doing it the other way?